Dockerfile

```diff
@@ -10,19 +10,16 @@ RUN apt update \
     && rm -rf /var/lib/apt/lists/*
 
 # Setup ROS workspace folders
-ENV ROS_WS /opt/ros_ws
-RUN mkdir -p $ROS_WS/src
+ENV ROS_WS=/opt/ros_ws
 WORKDIR $ROS_WS
-# Also create ecal_to_ros pkg-to-be folder
-RUN mkdir -p $ROS_WS/src/ecal_to_ros
 
-# -----------------------------------------------------------------------
+# Copy ROS2 msg files and bridge code over
+COPY ecal_to_ros/ros2/ src/ecal_to_ros/
```

Comment:
Any reason we changed from ADD to COPY? From my basic understanding, ADD is the same but can do more things. Is this just to future proof-it by using the least powerful command wherever possible?

Reply:
Yeap, is exacly that. `ADD` can decompress files automatically and has more functionalities. Here we only want to copy, so I always use COPY for the simplicty, an possible a bit less overhead?